lib/metadata/virtual_container_host.go

```diff
@@ -112,7 +112,7 @@ type VirtualContainerHostConfigSpec struct {
 
 	// FIXME: remove following attributes after change to launch through tether
 	// Networks represents mapping between nic name and network info object. For example: bridge: vmomi object
-	Networks map[string]*NetworkInfo `vic:"0.1" scope:"read-only" key:"networks"`
+	Networks map[string]*NetworkInfo `vic:"0.1" scope:"read-only" key:"networks2"`
```

Comment:
Currently there is a conflict with key names, with the old key names that vic-machine is using conflict with this one. This is just a workaround for now; the old keys in vic-machine will be discarded eventually once we have the tether running appliance finished.